mhkit/wave/io.py

```diff
@@ -112,30 +112,32 @@ def read_NDBC_file(file_name, missing_values=['MM',9999,999,99]):
 
 
 def ndbc_available_data(parameter='swden',
-                        number=None, 
+                        buoy_number=None, 
                         proxy=None):  
     '''
-    For a given parameter this will return a DataFrame of years, and 
-    station IDs and file names              
+    For a given parameter this will return a DataFrame of years, 
+    station IDs and file names that contain that parameter data.              
     
     Parameters
     ----------
-    parameter: string
-        'swden'	:	'Raw Spectral Wave Current Year Historical Data'
-    number: string
+    parameter: string (optional)
+        Current option is 'swden'	:	'Raw Spectral Wave Current Year Historical Data'
```

Comment:
This clutters the docstring in my opinion. I think this can go in the general description e.g. where you added `that contain that parameter data` we could either adjust that to say spectral wave density data. Or add another sentence that clarifies that `swden` is currently the only parameter supported.